includes/wf_crm_admin_help.inc

```diff
@@ -378,7 +378,17 @@ class wf_crm_admin_help {
     print '<p>' .
       t('If a matching case of the chosen type already exists for the client, it will be autofilled and updated.') .
       '</p><p>' .
-      t('Note: a case can also be autofilled by passing "case1", etc. in the url. To create a new case based on an existing case also append "newcase1=1" etc.') .
+      t('Note: a case can also be autofilled by passing "case1", etc. in the url.') .
+      '</p>';
+  }
+
+  public static function duplicate_case_status() {
+    print '<p>' .
+      t('This forces a new case to be created based on an existing case.') .
```

Comment:
The language here suggests that an existing case is required, but your code suggests it is not. It looks like the behavior if an existing case is not found would be to just create a new case from scratch (so it is not "forced" to find an existing case in order for the form to function). Is that right?